addons/website/static/src/js/content/menu.js

```diff
@@ -531,8 +536,11 @@ publicWidget.registry.menuDirection = publicWidget.Widget.extend({
      * @private
      */
     _onDropdownShow: function (ev) {
-        var $li = $(ev.target);
+        const $li = $(ev.target).closest('li');
```

Comment:
I think in the case of a `o_extra_menu_items`, the `nav-item` is not a `<li>` but a `<div>`, so this code doesn't works with the extra menu.
I tested the alignement of an extra menu dropdown on your branch. see =>

![Capture d’écran de 2023-03-13 11-43-14](https://user-images.githubusercontent.com/52911687/224680144-ad7481eb-0ab5-4f5b-bf58-c3473a508cdf.png)

Could you fix it please ;)

Reply:
Good catch indeed :+1: It's now fixed